ipatests/test_integration/test_installation.py

```diff
@@ -559,6 +560,21 @@ def install(cls, mh):
 
     def test_install_master(self):
         tasks.install_master(self.master, setup_dns=False)
+        """Test if schema-compat-entry-attribute is set
```

Comment:
Could you please move this test into a dedicated test function? It makes it easier to xfail it if necessary.